pkg/resource/route.go

```diff
@@ -67,9 +70,12 @@ func (gr *generatorRoute) GetName() string {
 func (gr *generatorRoute) expected() (runtime.Object, error) {
 	r := &routeapi.Route{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:        gr.GetName(),
-			Namespace:   gr.GetNamespace(),
-			Annotations: map[string]string{RouteOwnerAnnotation: "true"},
+			Name:      gr.GetName(),
+			Namespace: gr.GetNamespace(),
+			Annotations: map[string]string{
+				RouteTimeoutAnnotation: "30s",
```

Comment:
Let's not create constants for strings that we use only once and that are not part of our API.